subprojects/core/src/integTest/groovy/org/gradle/normalization/ConfigureRuntimeClasspathNormalizationIntegrationTest.groovy

```diff
@@ -339,6 +339,44 @@ class ConfigureRuntimeClasspathNormalizationIntegrationTest extends AbstractInte
         executedAndNotSkipped(project.customTask)
     }
 
+    @ToBeFixedForConfigurationCache(because = "classpath normalization")
+    def "properties files are normalized against changes to whitespace and comments"() {
+        def project = new ProjectWithRuntimeClasspathNormalization(true)
```

Comment:
Using an enum to name the value of this `boolean` parameter would make the code more readable.

Reply:
Good point.  Changed.